### PR TITLE
update for CAOM 2.3

### PR DESCRIPTION
**Pull request by pdowler**

### CODE REVIEW

File: caom2-meta-server/src/main/java/ca/nrc/cadc/caom2/meta/MetaQueryRunner22.java [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+import java.net.URISyntaxException;
+
+/**
+ * MetaQueryRunner hard coded to output CAOM-2.2 documents.
+ * 
+ * @author pdowler
+ */
+public class MetaQueryRunner22 implements JobRunner
+{
+    private static final Logger log = Logger.getLogger(MetaQueryRunner.class);
```
Comment (andamian) [inline caom2-meta-server/src/main/java/ca/nrc/cadc/caom2/meta/MetaQueryRunner22.java]: Not MetaQueryRunner22?

File: caom2-meta-server/src/main/java/ca/nrc/cadc/caom2/meta/MetaQueryRunner22.java [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+        {
+            sendError(ex, ex.getMessage(), ex.getResponseCode());
+        }
+        catch(Throwable t)
+        {
+            if ( ThrowableUtil.isACause(t, InterruptedException.class) )
+            {
+                try
+                {
+                    ep = jobUpdater.setPhase(job.getID(), ExecutionPhase.QUEUED, ExecutionPhase.EXECUTING, new Date());
```
Comment (andamian) [inline caom2-meta-server/src/main/java/ca/nrc/cadc/caom2/meta/MetaQueryRunner22.java:279]: QUEUED->EXECUTING?

File: caom2-tap/src/main/java/ca/nrc/cadc/caom2ops/mapper/ArtifactMapper.java [hunk 3]
```diff
@@ -140,19 +140,25 @@ public Artifact mapRow(List<Object> data, DateFormat dateFormat)
...
+            artifact.contentChecksum = Util.getURI(data, map.get("caom2:Artifact.contentChecksum"));
 
             Date lastModified = Util.getDate(data, map.get("caom2:Artifact.lastModified"));
             Date maxLastModified = Util.getDate(data, map.get("caom2:Artifact.maxLastModified"));
-
             Util.assignLastModified(artifact, lastModified, "lastModified");
             Util.assignLastModified(artifact, maxLastModified, "maxLastModified");
+            
+            URI metaChecksum = Util.getURI(data, map.get("caom2:Plane.metaChecksum"));
+            URI accMetaChecksum = Util.getURI(data, map.get("caom2:Plane.accMetaChecksum"));
```
Comment (andamian) [inline caom2-tap/src/main/java/ca/nrc/cadc/caom2ops/mapper/ArtifactMapper.java]: Change Plane with Artifact in the schema.

File: caom2-meta-server/src/main/java/ca/nrc/cadc/caom2/meta/MetaQueryRunner22.java (hunk 1 above)
Comment (pdowler) [reply caom2-meta-server/src/main/java/ca/nrc/cadc/caom2/meta/MetaQueryRunner22.java]: fixing

File: caom2-meta-server/src/main/java/ca/nrc/cadc/caom2/meta/MetaQueryRunner22.java (hunk 2 above)
Comment (pdowler) [reply caom2-meta-server/src/main/java/ca/nrc/cadc/caom2/meta/MetaQueryRunner22.java:279]: this is correct. The UWS servlet puts it from pending to queued and job puts it to executing

File: caom2-tap/src/main/java/ca/nrc/cadc/caom2ops/mapper/ArtifactMapper.java (hunk 3 above)
Comment (pdowler) [reply caom2-tap/src/main/java/ca/nrc/cadc/caom2ops/mapper/ArtifactMapper.java]: good catch - fixing

**Review by andamian — APPROVED** [review]

**Pull request merged**